tests/coco_mock.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020–2021 Cirq on IQM developers
```

Comment:
This should be a conftest.py and use pytest fixtures.

Reply:
done